nsqd/options.go

```diff
@@ -90,6 +92,8 @@ func NewOptions() *Options {
 	return &Options{
 		ID:        defaultID,
 		LogPrefix: "[nsqd] ",
+		LogLevel:  "info",
+		logLevel:  2,
```

Comment:
I think putting the `logLevel` int default in `nsqd.NewOptions()` isn't needed, because it's set from the string in `nsqd.New()`. (The same goes for nsqadmin, but you didn't set the logLevel int default in `nsqadmin.NewOptions()`.)

But this doesn't hurt (except a tiny bit of code redundancy), and I think it may be fine to wait to fix it in my planned follow-up PR.